examples/vertx-opentracing/README.md

```diff
@@ -0,0 +1,153 @@
+
+    oc get route order-manager
+
+To access the OpenShift console, type in `oc status` and check the address on the first line of the output. The username and
+password is `developer` / `developer`
+
+At the end of the process, you should be able to get a JSON as response to this command:
+
+    curl -X POST -d '{"accountId":"fred","itemId":"laptop","quantity":2}' http://order-manager-order-manager.<your OpenShift IP>.xip.io/orders
+
+## Setting it up - Option 2 - Manual
+
+To run this, you'll need an OpenShift cluster (local or otherwise), with Hawkular APM running on it. This can be accomplished
+with the following commands on Fedora 24 (or later):
+
+    sudo iptables -F
+    oc cluster up
+    oc create -f https://raw.githubusercontent.com/jboss-dockerfiles/hawkular-apm/master/openshift-templates/hawkular-apm-server-deployment.yml
+
+Make note of the UI address provided by the `oc cluster up` command.
+Once Hawkular APM is up and running, the example as a whole needs to be build and the microservices needs to be added to OpenShift.
+The following commands can be executed in order to accomplish that:
+
+    mvn clean install
+    ## account manager
+    cd account-manager
+    oc new-build --binary --name=account-manager -l app=account-manager
+    oc start-build account-manager --from-dir=. --follow
+    oc new-app account-manager HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=account-manager
+    cd ..
+    ## inventory manager
+    cd inventory-manager
+    oc new-build --binary --name=inventory-manager -l app=inventory-manager
+    oc start-build inventory-manager --from-dir=. --follow
+    oc new-app inventory-manager HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=inventory-manager
+    cd ..
+    ## order log
+    cd order-log
+    oc new-build --binary --name=order-log -l app=order-log
+    oc start-build order-log --from-dir=. --follow
+    oc new-app order-log HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=order-log
+    cd ..
+    ## order manager
+    cd order-manager
+    oc new-build --binary --name=order-manager -l app=order-manager
+    oc start-build order-manager --from-dir=. --follow
+    oc new-app order-manager HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=order-manager
+    oc expose service order-manager
+    cd ..
+
+Login to the OpenShift UI at the address given by the `oc cluster up` command and wait until all services are "blue", meaning
+that they are ready. At this point, you can execute the following command to test, replacing the hostname by the appropriate value,
+which can be seen on the OpenShift UI:
+
+    curl -X POST -d '{"accountId":"fred","itemId":"laptop","quantity":2}' http://order-manager-myproject.<your OpenShift IP>.xip.io/orders
+
```

Comment:
Wondering whether it needs the `oc get route order-manager` text here as well? Ideally it would be good to have it in one place - may be the text for trying on openshift could be in one place - but then the user would need to be directed from the two places?